deepchecks/vision/checks/performance/single_dataset_scalar_performance.py

```diff
@@ -0,0 +1,148 @@
+        metric_result = self.metric.compute()
+        if self.reduce is not None:
+            if isinstance(metric_result, float):
+                warnings.warn(SyntaxWarning('Metric result is already scalar, skipping reduce operation'))
+                result_value = metric_result
+            else:
+                result_value = float(self.reduce(metric_result))
+        elif isinstance(metric_result, float):
+            result_value = metric_result
+        else:
+            raise DeepchecksValueError(f'The metric {self.metric.__class__} return a non-scalar value, '
+                                       f'please specify a reduce function or choose a different metric')
+
+        metric_name = self.metric_name if hasattr(self, 'metric_name') else self.metric.__class__.__name__
+        if hasattr(self, 'reduce_name'):
+            reduce_name = self.reduce_name
+        elif self.reduce is None:
+            reduce_name = None
+        else:
+            reduce_name = self.reduce.__name__
+
+        result_dict = {'score': result_value,
+                       'metric': metric_name,
+                       'reduce': reduce_name}
+        return CheckResult(result_dict)
+
+    def add_condition_greater_than(self, threshold: float) -> ConditionResult:
+        """Add condition - the result is greater than the threshold."""
+        def condition(check_result):
+            if check_result['score'] > threshold:
+                return ConditionResult(ConditionCategory.PASS)
+            else:
+                details = f'The score is not greater than {threshold}'
+                return ConditionResult(ConditionCategory.FAIL, details)
+        return self.add_condition(f'Score is greater than {threshold}', condition)
+
+    def add_condition_greater_equal_to(self, threshold: float) -> ConditionResult:
+        """Add condition - the result is greater than the threshold."""
+
+        def condition(check_result):
+            if check_result['score'] >= threshold:
+                return ConditionResult(ConditionCategory.PASS)
+            else:
+                details = f'The score is not greater than or equal to {threshold}'
+                return ConditionResult(ConditionCategory.FAIL, details)
+
+        return self.add_condition(f'Score is greater than or equal to {threshold}', condition)
+
+    def add_condition_less_than(self, threshold: float) -> ConditionResult:
+        """Add condition - the result is greater than the threshold."""
+        def condition(check_result):
+            if check_result['score'] < threshold:
+                return ConditionResult(ConditionCategory.PASS)
+            else:
+                details = f'The score is not less than {threshold}'
+                return ConditionResult(ConditionCategory.FAIL, details)
+        return self.add_condition(f'Score is less than {threshold}', condition)
+
+    def add_condition_less_equal_to(self, threshold: float) -> ConditionResult:
```

Comment:
is there a real reason for less equal? isn't less enough? just that equal for floats have no real meaning (same comment for greater equal)

Reply:
@JKL98ISR I think the users will feel more comfortable with this, about the syntax I thought we decided to change the negative conditions to positive ones